### PR TITLE
Update polymer-cli docs to mention /etc/hosts file

### PR DESCRIPTION
**Pull request by JimPanic**
Regarding https://github.com/Polymer/polymer-cli/issues/312

### CODE REVIEW

**Comment by JimPanic** [comment]
I signed it!

**Comment by arthurevans** [comment]
@justinfagnani can you take a look at this? Is this correct, actionable, portable? I haven't used this particular feature.

**Comment by justinfagnani** [comment]
I'm not sure sure about this. It's very UNIX-centric, and documents something arguably outside the scope of the CLI: you're local computer needs to have it's networking properly configured to reach a host.

**Comment by justinfagnani** [comment]
I do think the quotes around 'test' need to be removed.

**Comment by JimPanic** [comment]
> I'm not sure sure about this. It's very UNIX-centric, and documents something arguably outside the scope of the CLI: you're local computer needs to have it's networking properly configured to reach a host.

This is very much the scope of the CLI: it is a feature to listen to a hostname other than the IP and a port alone. For that hostname to work it needs to resolve to any local IP. This has also nothing to do with having networking configured or not, this works without any wifi or cable connected as well.

You are right, the `/etc/hosts` was quite UNIX-centric, I changed it now.

> I do think the quotes around 'test' need to be removed.

They might be misinterpreted on some systems to be part of the hostname. You should not need the quotes here anyways.

**Comment by JimPanic** [comment]
Looking at the original issue, they *are* misinterpreted by Windows 10 to be part of the argument apparently.

**Comment by FredKSchott** [comment]
+1 for removing the quotes

Instead of adding an entirely  new NOTE section can we just rephrase the paragraph above to mention that more configuration is needed by the user? Something like:

>If you've configured a custom hostname on your machine, you can serve from it with the `--hostname` argument (e.g. app project demo is available at `http://test:8080`):

>```
>polymer serve --hostname test`
>```

**Comment by JimPanic** [comment]
FYI from the other original issue in polymer-cli:

> I just tested it on a colleague's Windows machine: this is an issue with `cmd.exe`. PowerShell works the same as any UNIX shell in this regard and strips the single and double quotes. `cmd.exe` strips double quotes but not single quotes.

File: app/1.0/docs/tools/polymer-cli.md [hunk 1]
```diff
@@ -266,10 +266,10 @@ Serve from port 3000:
 
     polymer serve --port 3000
 
-Serve from hostname `test` (e.g. app project demo is available at
-`http://test:8080`):
+If you configured a custom hostname on your machine, polymer can serve it with the
```
Comment (FredKSchott) [inline app/1.0/docs/tools/polymer-cli.md]: Small nit: `If you have configured`

**Comment by FredKSchott** [comment]
+1 from me with the one small Grammar Nazi fix

File: app/1.0/docs/tools/polymer-cli.md (hunk 1 above)
Comment (JimPanic) [inline app/1.0/docs/tools/polymer-cli.md]: Ay!

**Comment by JimPanic** [comment]
@FredKSchott Thanks for catching that!

I asked a colleague as well who is a native speaker unlike me and we ended up both being completely confused by if sentences and their rules. lol.

**Comment by FredKSchott** [comment]
Haha no problem, to be honest I'm not even 100% sure if either is "correct" but I think the change reads more formally and less casually which is a good thing imo.

LGTM, @arthurevans would you do the honors of giving a final look-over and merging?

File: app/1.0/docs/tools/polymer-cli.md [hunk 2]
```diff
@@ -266,10 +266,10 @@ Serve from port 3000:
 
     polymer serve --port 3000
 
-Serve from hostname `test` (e.g. app project demo is available at
-`http://test:8080`):
+If you have configured a custom hostname on your machine, polymer can serve it with the
```
Comment (arthurevans) [inline app/1.0/docs/tools/polymer-cli.md]: polymer => Polymer CLI

File: app/1.0/docs/tools/polymer-cli.md [hunk 3]
```diff
@@ -266,10 +266,10 @@ Serve from port 3000:
...
+`--hostname` argument (e.g. app project demo is available at `http://test:8080`):
```
Comment (arthurevans) [inline app/1.0/docs/tools/polymer-cli.md]: "e.g." => "for example,"

**Comment by arthurevans** [comment]
Minor nits to match doc style. Otherwise LGTM.

**Comment by JimPanic** [comment]
There we go, incorporated the changes. 👍

**Comment by arthurevans** [comment]
:dart: Awesome. Thanks!

**Comment by FredKSchott** [comment]
@JimPanic woot! Thanks for working to get this merged!

**Pull request merged**